src/main/java/de/azapps/kafkabackup/common/topic/restore/RestoreArgsWrapper.java

```diff
@@ -37,7 +40,8 @@
   public static final String KAFKA_BOOTSTRAP_SERVERS = "kafka.bootstrap.servers";
 
   public static final String RESTORE_DRY_RUN = "restore.dryRun";
-  public static final String RESTORE_TOPIC_LIST = "restore.topicList";
+  public static final String RESTORE_TOPIC_LIST_MODE = "restore.topicsList.mode";
```

Comment:
Hm. Why not have `restore.topicDenyList` and `restore.topicAllowList` and both take regexps? Like for burrow: https://github.com/getdreams/dreams2-burrow/blob/master/config/burrow-dreams2-stage.toml#L28-L29
And then just use both to filter? (the order they are applied shouldn't matter, unless I'm confused)

Reply:
Just remember that an empty allowList means `.*`

Reply:
right, sounds like a better solution - I'll change this!